ajax/libs/cloudinary-jquery/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "cloudinary-jquery",
+  "description": "Cloudinary Client Side JS library. Cloudinary streamlines your web application’s image manipulation needs. Cloudinary's cloud-based servers automate image uploading, resizing, cropping, optimizing, sprite generation and more.",
+  "filename": "cloudinary-jquery.min.js",
+  "author": {
+    "name": "Cloudinary",
+    "email": "info@cloudinary.com",
+    "homepage": "https://cloudinary.com"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cloudinary/pkg-cloudinary-jquery.git"
```

```suggestion
    "url": "https://github.com/cloudinary/cloudinary_js.git"
```

I believe the repo for this package has now changed, based on https://www.npmjs.com/package/cloudinary-jquery